at-tanvo/D3-Exercise1.rb

```diff
@@ -0,0 +1,22 @@
+loop do
+  puts "Enter Number"
+  a = gets.to_i
+  if a>=9 && a<=10 
+    puts "A"
+    break
+  elsif a>=8 && a<9
+    puts "B"
+    break
+  elsif a>=7 && a<8
+    puts "C"    
+    break
+  elsif a>=6 && a<7
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/c8f8a9c80a1053165f228ac8903bc3e0e0d677eb/at-tanvo/D3-Exercise1.rb#L13"></span>Layout/SpaceAroundOperators: Surrounding space missing for operator <code>&lt;</code>. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>